hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/DataTieringManager.java

```diff
@@ -45,6 +45,8 @@
 @InterfaceAudience.Private
 public class DataTieringManager {
   private static final Logger LOG = LoggerFactory.getLogger(DataTieringManager.class);
+  public static final String DATA_TIERING_ENABLED_KEY = "hbase.hstore.datatiering.enable";
```

Comment:
Since it is a cluster-wide (or region server-level) configuration, I think the format should be something like `hbase.regionserver.*`, although I'm not entirely sure. Additionally, could we use a better name for the variable to indicate that this is a cluster-wide configuration? like, `GLOBAL_DATA_TIERING_ENABLED_KEY`?